WiseBuy/View Controllers/RegisterViewController.m

```diff
@@ -75,17 +75,22 @@ - (void)resetFields {
     self.usernameField.text = @"";
     self.passwordField.text = @"";
     self.emailField.text = @"";
-    self.profileImage.image = [UIImage imageNamed: @""];
+    self.profileImage.image = [UIImage imageNamed: @"person.circle.fill"];
 }
 
 -(BOOL) inputFieldsAreEmpty {
     return [self.firstNameField.text isEqual:@""]  || [self.lastNameField.text isEqual:@""]  || [self.emailField.text isEqual:@""] || [self.usernameField.text isEqual:@""] || [self.passwordField.text isEqual:@""];
 }
 
--(BOOL)inputFieldsContainSpacesOrNewLines {
-    NSCharacterSet *charSet = [NSCharacterSet whitespaceAndNewlineCharacterSet];
+- (BOOL)inputFieldsContainSpacesOrNewLines {
+    NSCharacterSet *set = [NSCharacterSet whitespaceAndNewlineCharacterSet];
```

Comment:
Small nit--maybe `whitespaceAndNewlineSet` or `whitespaceAndNewlines` to distinguish it from `newLineSet`